meson_options.txt

```diff
@@ -0,0 +1,4 @@
+option('build_docs',
+	type: 'boolean',
+	value: false,
+	description: 'Build reference documentation (needs HotDoc)')
```

Comment:
There's an equivalent to the `EXPORTABLE_EGL` CMake option missing here. I mentioned somewhere before (maybe here, can't remember) that the option should be made no-op and the EGL-oriented exportable interfaces always built and installed.

If necessary for whatever aspect of portability and compliance, then a similar no-op `EXPORTABLE_EGL`-equivalent option should be added here. CMakeLists.txt should also be modified to build and install the EGL-oriented interfaces unconditionally.

Reply:
Instead of making it a no-op, I would rather remove the `option()` from `CMakeLists.txt`, because CMake will still accepts random undeclared `-DFOO=` options in the command line and will warn about them: existing build scripts won't break, but they will get a warning.

WDYT?

Reply:
Sounds fine to me.